src/main/java/org/mastodon/views/trackscheme/display/EditFocusVertexLabelAction.java

```diff
@@ -180,6 +180,12 @@ public void updateHeaderSize( final int width, final int height )
 		headerHeight = height;
 	}
 
+	protected void changeLabel( TrackSchemeVertex vertex, String label )
+	{
+		vertex.setLabel( label );
+		undoPointMarker.setUndoPoint();
```

Comment:
We probably should make an undo point only once, after the whole branch has been renamed.
Here we store an undo point for every vertex being renamed.

Reply:
There a little confusion here going on. This method is not used to rename the branch labels. This is the method that is used to rename a single individual spot in the normal TrackScheme window, when pressing enter. Here we want an undo point after each individually renamed spot.

*(The source code is changed here to make it easier for me to extend this class for the Branch TrackScheme. Notice how the method is protected. It is overwritten in BranchTrackSchemeEditLabelAction.)*